frontend/src/app/commonComponents/Checkboxes.tsx

```diff
@@ -87,7 +87,7 @@ const Checkboxes = (props: Props) => {
   return (
     <div
       className={classnames(
-        "usa-form-group",
+        "usa-form-group padding-0",
```

Comment:
noticed some extra padding in the AOE form so adding this to fix

<img width="702" alt="Screenshot 2024-06-05 at 3 59 32 PM" src="https://github.com/CDCgov/prime-simplereport/assets/29645040/a1b5dcbf-7f7c-463f-9078-440ec6935a12">

Reply:
OO thank you for fixing this!